package.json

```diff
@@ -34,7 +40,7 @@
     "off chain"
   ],
   "devDependencies": {
-    "ganache-cli": "^6.1.8",
+    "ganache-cli": "6.1.8",
```

Comment:
The reason for this was noted in the commit message: https://stackoverflow.com/a/53413796/10047840